### PR TITLE
added filter by gojq

### PR DESCRIPTION
**Pull request by yseto**
Incorporate filters by gojq.
Useful since results are often parsed with jq

### CODE REVIEW

File: format/format.go [hunk 1]
```diff
@@ -46,3 +53,65 @@ func ISO8601Extended(t time.Time) string {
...
+			_, err = fmt.Fprintln(outStream, JSONMarshalIndent(v, "", "    "))
+			if err != nil {
+				return err
+			}
+		}
+	}
+	return nil
+}
+
+// copy from https://github.com/cli/cli/blob/d21d388b8dc10c8f04187c3afa6e0b44f0977c65/pkg/export/template.go#L110-L127
```
Comment (Arthur1) [inline format/format.go:99]: If this is a copy, a notice of copyright and license may be needed.

File: hosts/app.go [hunk 2]
```diff
@@ -45,14 +46,16 @@ func (ha *hostApp) findHosts(param findHostsParam) error {
 	}
 
 	switch {
+	case param.format != "" && ha.jq != "":
+		return fmt.Errorf("--format and --jq are exclusive use.")
```
Comment (Arthur1) [inline hosts/app.go:50]: How about the following expression?

```
--format and --jq options are incompatible.
```

File: aws_integrations/app.go [hunk 3]
```diff
@@ -11,6 +11,7 @@ import (
 type awsIntegrationsApp struct {
 	client    mackerelclient.Client
 	outStream io.Writer
+	jq        string
```
Comment (Arthur1) [inline aws_integrations/app.go:14]: The variable name `jq` is unclear. How about `jqFilter`?

File: format/format.go [hunk 4]
```diff
@@ -24,9 +27,13 @@ type Host struct {
 }
 
 // PrettyPrintJSON output indented json via stdout.
```
Comment (Arthur1) [inline format/format.go:29]: `func PrettyPrintJSON` may not output indented json if filtered value is single.

**Comment by yseto** [comment]
@Arthur1
Thanks for your review. i fixed them.

File: format/format.go [hunk 5]
```diff
@@ -23,10 +24,15 @@ type Host struct {
 	IPAddresses   map[string]string `json:"ipAddresses,omitempty"`
 }
 
-// PrettyPrintJSON output indented json via stdout.
-func PrettyPrintJSON(outStream io.Writer, src interface{}) error {
-	_, err := fmt.Fprintln(outStream, JSONMarshalIndent(src, "", "    "))
-	return err
+// PrettyPrintJSON output json via stdout
```
Comment (Arthur1) [inline format/format.go:27]: I suggest the following expression as a comment.

```
PrettyPrintJSON outputs JSON or filtered result by jq query via stdout.
```

**Review by Arthur1 — APPROVED** [review]
It seems good to us. Please have a look at the minor suggestion about comments.

**Pull request merged**